go/host/rpc/clientapi/client_api_obscurotest.go

```diff
@@ -20,24 +18,16 @@ func NewTestAPI(host host.Host) *TestAPI {
 	}
 }
 
-// GetID returns the ID of the host.
-func (api *TestAPI) GetID() gethcommon.Address {
-	return api.host.Config().ID
-}
-
-// GetHeadBlockHeader returns the current head block's header.
-func (api *TestAPI) GetHeadBlockHeader() *types.Header {
```

Comment:
Didn't actually need the block header, just its height. So replaced this with `BlockNumber()`, below.